airbyte-integrations/connectors/source-recharge/source_recharge/api.py

```diff
@@ -174,8 +174,14 @@ class Onetimes(IncrementalRechargeStream):
 class Orders(IncrementalRechargeStream):
     """
     Orders Stream: https://developer.rechargepayments.com/v1-shopify?python#list-orders
+    Using old API version to avoid schema changes and loosing email, first_name, last_name columns, because in new version it not present
     """
 
+    def request_headers(
+        self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None
+    ) -> Mapping[str, Any]:
+        return {"x-recharge-version": OLD_API_VERSION}
```

Comment:
I think it's better to use a specific variable name format such as `API_VERSION_2021_01`. This approach will help us avoid confusion as we continue to introduce multiple versions of API for streams in the future.